manager/integration/tests/common.py

```diff
@@ -212,6 +233,122 @@ def get_compatibility_test_image(cli_v, cli_minv,
                                      data_v, data_minv)
 
 
+def generate_random_data(count):
+    return ''.join(random.choice(string.ascii_lowercase + string.digits)
+                   for _ in range(count))
+
+
+def volume_read(dev, start, count):
+    r_data = ""
+    fdev = open(dev, 'rb')
+    if fdev is not None:
```

Comment:
Don't need to check None or not. If it's none, it will error out later. Or you can use `assert` to make sure it's not None.